test/e2e/conftest.py

```diff
@@ -0,0 +1,197 @@
+
+    client = boto3.client('cloudformation', region_name=testing_env_variables['REGION'])
+    response = client.describe_stacks(StackName=testing_env_variables['MIE_STACK_NAME'])
+    outputs = response['Stacks'][0]['Outputs']
+
+    for output in outputs:
+        resources[output["OutputKey"]] = output["OutputValue"]
+
+    assert "WorkflowApiEndpoint" in resources
+    assert "DataplaneApiEndpoint" in resources
+
+    api_endpoint_regex = ".*.execute-api."+testing_env_variables['REGION']+".amazonaws.com/api/.*"
+
+    assert re.match(api_endpoint_regex, resources["WorkflowApiEndpoint"])
+
+    response = client.describe_stacks(StackName=resources["OperatorLibraryStack"])
+    outputs = response['Stacks'][0]['Outputs']
+    for output in outputs:
+        resources[output["OutputKey"]] = output["OutputValue"]
+
+    expected_resources = ['WorkflowApiRestID', 'DataplaneBucket', 'WorkflowCustomResourceArn', 'TestStack', 'MediaInsightsEnginePython37Layer', 'AnalyticsStreamArn', 'DataplaneApiEndpoint', 'WorkflowApiEndpoint', 'DataplaneApiRestID', 'OperatorLibraryStack', 'PollyOperation', 'ContentModerationOperationImage', 'GenericDataLookupOperation', 'comprehendEntitiesOperation', 'FaceSearch', 'FaceSearchOperationImage', 'MediainfoOperationImage', 'TextDetection', 'TextDetectionOperationImage', 'CreateSRTCaptionsOperation', 'TranscribeOperation', 'ContentModeration', 'WebCaptionsOperation', 'WebToVTTCaptionsOperation', 'CelebRecognition', 'LabelDetection', 'FaceDetection', 'PersonTracking', 'MediaconvertOperation', 'FaceDetectionOperationImage', 'MediainfoOperation', 'ThumbnailOperation', 'TechnicalCueDetection', 'CreateVTTCaptionsOperation', 'CelebrityRecognitionOperationImage', 'TranslateOperation', 'comprehendPhrasesOperation', 'WebToSRTCaptionsOperation', 'shotDetection', 'LabelDetectionOperationImage', 'StackName']
+
+    assert set(resources.keys()) == set(expected_resources)
+
+    return resources
+
+
+# This fixture uploads the sample media objects for testing.
+
+
+@pytest.fixture(scope='session', autouse=True)
+def upload_media(testing_env_variables, stack_resources):
+    print('Uploading Test Media')
+    s3 = boto3.client('s3', region_name=testing_env_variables['REGION'])
+    # Upload test media files
+    # s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_TEXT'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_TEXT'])
+    # s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_JSON'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_JSON'])
+    # s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_AUDIO'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_AUDIO'])
+    # s3.upload_file(testing_env_variables['MEDIA_PATH'] + testing_env_variables['SAMPLE_IMAGE'], stack_resources['DataplaneBucket'], 'upload/' + testing_env_variables['SAMPLE_IMAGE'])
```

Comment:
Why no image test?

Reply:
We don't have any published applications that explicitly support images at this point. If I remember correctly, we decided to scope down CAS to video only. 

Once we expand support for CAS or add a new application that uses images, it will be simple enough to add an image workflow test.